tests/framework/test_values.py

```diff
@@ -14,27 +19,29 @@
 
 
 @pytest.fixture
-def static_step():
+def static_step() -> Callable[[pd.Index[int]], pd.Series[pd.Timedelta]]:
     return lambda idx: pd.Series(pd.Timedelta(days=6), index=idx)
 
 
 @pytest.fixture
-def variable_step():
+def variable_step() -> Callable[[pd.Index[int]], pd.Series[pd.Timedelta]]:
     return lambda idx: pd.Series(
         [pd.Timedelta(days=3) if i % 2 == 0 else pd.Timedelta(days=5) for i in idx], index=idx
     )
 
 
 @pytest.fixture
-def manager(mocker):
+def manager(mocker: pytest_mock.MockFixture) -> ValuesManager:
```

Comment:
nit: it's less typing to `from pytest_mock import MockFixture` once at the top

Reply:
Good call